### ISSUE TITLE
ASL Eye Tracker data file support

### ISSUE THREAD

**Issue by munoztd0** [bachlab/PsPM#141]
### Feature Description

Hello,

After following your PsPM course series i wanted to try it with my own dataset but it looks like my ASL Eye Tracker data file is not supported by PsPM import. So I just put it out there to see if other people would also need a feature update on that type of datafile.

Thanks.

**Comment by irojkov-ph** [comment]
Hi David,

Thank you very much for opening this feature request, that helps us to improve PsPM.
I create a poll here to know how many people are interested in such a feature ;)

### Would you like PsPM to be able to import ASL Eye Tracker data files ?
[![](https://api.gh-polls.com/poll/01EFEEDPWKQK424HKE5C1HRYS1/Yes)](https://api.gh-polls.com/poll/01EFEEDPWKQK424HKE5C1HRYS1/Yes/vote)
[![](https://api.gh-polls.com/poll/01EFEEDPWKQK424HKE5C1HRYS1/No)](https://api.gh-polls.com/poll/01EFEEDPWKQK424HKE5C1HRYS1/No/vote)

**Comment by irojkov-ph** [comment]
Hi again David,
In case we have to implement it, may I ask you to provide us a sample data file and some documentation (if you have some) please ?
Thank you in advance ;) 

P.S.: I ask you that because I haven't found anything on ASL's website.

**Comment by munoztd0** [comment]
Yeah of course here you go, [https://drive.switch.ch/index.php/s/9KscHeL5kyyaax3](url).

I put a raw .eyd (ASL) file and a converted to .mat file for convenience.

As for the documentation, I'm afraid I don't really find anything neither. I just know our lab has been using the Ilab toolbox (ilabConvertASL()) to convert it to a readable .mat format. But I think it's relatively similar to an exported biopac datafile.

Thanks for you help and interest, don't hesitate if you need anything else.

**Comment by dominikbach** [comment]
Not sure what happened here but we don't have access to the test data.
